### DISCUSSION TITLE
Kimai creating lots of Files on webspace that go away after reloading cache

### DISCUSSION THREAD

**Discussion by HubiKaehn** [kimai/kimai#5167] [Questions & Answers]
Hi,
Kimai is working great but it is creating a lot of files (log or cache files). I manually have to reload the Kimai cache ones in a while or it uses up a lot space. I use IONOS and have a file number limit on my webspace rather than a file size limit:

Before reloading cache:
<img width="334" alt="before reloading cache" src="https://github.com/user-attachments/assets/cf0ef0c7-bc2c-4493-9dd3-6ca30f619c1a">
After reloading cache:
<img width="329" alt="after reloading cache" src="https://github.com/user-attachments/assets/d8153092-a352-4c61-89a2-175667928541">

Is there a way to make it clear that cache automatically or not create as many files? I have different websites on that webspace as well that a subjectively much larger but don't use up nearly as many files.

Thank you for your help!

**Comment by kevinpapst** [comment]
That's an issue that is caused by Symfony, the framework that Kimai uses under the hood.

I know it sucks. Unfortunately I don't know why that happens or how to fix it. Usually doesn't cause problems, unless you use certain hosting environments...

 If anyone knows how to fix that, I'd be happy to hear about it as well and integrate any solution!

**Reply by HubiKaehn** [reply]
Okay, I will just wait, regulary reload the cache and wait for someone to come up with a solution :)

**Comment by ecoinch** [comment]
Hi, can you share how you are reloading cache for Kimai as it is not working for me.

**Reply by kevinpapst** [reply]
> as it is not working for me.

What does that mean? Share what you do. Which part of the documentation is not sufficient?

**Reply by ecoinch** [reply]
On cPanel, navigating to LiteSpeed Web Cache Manager (on A2 Hosting). 

![image](https://github.com/user-attachments/assets/38195816-c79a-45a5-b657-717b6570fa92)

From there, click on Flush All:
![image](https://github.com/user-attachments/assets/59d7a03a-6a63-4965-a8c8-f4d2413cde4e)
